mail/api.py

```diff
@@ -20,18 +20,24 @@
 """
 import logging
 import re
+import requests
```

Comment:
There are imports in this file but there is no direct usage. I think we should remove them,